### DISCUSSION TITLE
[Feature Request] Toggle Event Thing

### DISCUSSION THREAD

**Discussion by Yoshi1203** [CapsAdmin/pac3#419] [Ideas]
I am trying to make it so I can toggle my keyboard to become a set of binds for customizations, and I am doing this thing where one button toggles a command "toggle" and then under a command event that finds "toggle," a bunch of keys become available through the button event to set their own commands, i.e. "made_up_customization," and then further elsewhere, it will look for whether those commands are on to set a customization. This is too difficult to set up to be worth it and furthermore only works sometimes do to who knows what. An event for toggling commands would help much, i.e. something like "toggle 'made_up_customization.'" Hopefully you can respond to this and also read it since I'm no good at explaining things.

**Comment by Vuthakral** [comment]
PAC already has a toggle function, albeit not quite what you are asking.

if you make a command bind such as: 'bind <key> "pac_event yourcommandname 2" will make it a toggle bind. So any events set to command and find "yourcommandname" will toggle on/off.

As for your specific part of the question, I'm not sure it's supported in PAC right now, where you can press one button and then another as a key-combination toggle, if I'm understanding what you're asking.

**Comment by Yoshi1203** [comment]
> if you make a command bind such as: 'bind "pac_event yourcommandname 2" will make it a toggle bind. So any events set to command and find "yourcommandname" will toggle on/off.

>  I tried this but the function seems to be not working for some reason. .-.
